### ISSUE TITLE
False positive: Property 'default' does not exist on type 'unknown' for import.meta.glob

### ISSUE THREAD

**Issue by janosh** [pheuter/svelte-check-rs#92]
When using \`import.meta.glob\` with \`{ eager: true }\` and destructuring the \`default\` export, svelte-check-rs types the module as \`unknown\` instead of recognizing the module structure.

## Reproduction

```svelte
<h1>Test</h1>

{#each Object.entries(import.meta.glob(`./*.md`, { eager: true })) as [path, { default: MdFile }] (path)}
  <MdFile />
{/each}
```

## Error

```
Error: Property 'default' does not exist on type 'unknown'. (ts(TS2339))
```

## Workaround

Type assertion:
```ts
as [string, { default: typeof SvelteComponent }]
```

svelte-check-rs: 0.7.3
